cmd/manager/main.go

```diff
@@ -4,19 +4,53 @@ import (
 	"flag"
 	"os"
 	"runtime"
+	"strconv"
 
 	sdkVersion "github.com/operator-framework/operator-sdk/version"
+	"github.com/storageos/cluster-operator/internal/pkg/admission"
+	"github.com/storageos/cluster-operator/internal/pkg/admission/scheduler"
 	"github.com/storageos/cluster-operator/pkg/apis"
 	"github.com/storageos/cluster-operator/pkg/controller"
+	"github.com/storageos/cluster-operator/pkg/storageos"
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	corev1 "k8s.io/api/core/v1"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+	webhookAdmission "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
 var log = logf.Log.WithName("storageos.setup")
 
+const (
+	// Env vars.
+	enableSchedulerEnvVar = "ENABLE_SCHEDULER"
+	podNamespaceEnvVar    = "POD_NAMESPACE"
+
+	// StorageOS provisioner names.
+	csiProvisioner    = "storageos"
+	nativeProvisioner = "kubernetes.io/storageos"
+
+	// operatorNameLabel is the "name" label in the operator's deployment
+	// config. This is needed for proper operator pod selection by webhook
+	// service.
+	// NOTE: If the operator's deployment changes the name label, this must be
+	// updated.
+	operatorNameLabel = "storageos-cluster-operator"
+
+	// podSchedulerWebhookName is a fully qualified name of the pod scheduler
+	// admission webhook.
+	podSchedulerWebhookName = "podscheduler.storageos.com"
+	// podSchedulerResourceName is the name of webhook server, service,
+	// mutatingwebhookconfig and other related resources.
+	podSchedulerResourceName = "pod-scheduler-webhook"
```

Comment:
I think it'll be better to have storageos name in this. Will make it easy to identify the resources that belong to storageos.
Maybe `storageos-scheduler-webhook`.